app/importers/reading/somerville_mega_reading_importer.rb

```diff
@@ -0,0 +1,67 @@
+class SomervilleMegaReadingImporter
```

Comment:
@edavidsonsawyer I thought this might be helpful for talking more concretely about what it might look like to write a reading importer.  This is placeholder for talking about that, with the idea that this might be able to combine parts of `MegaReadingImporter` with some kind of "Fetcher" class from https://github.com/studentinsights/studentinsights/pull/2442.

Reply:
Nice! I've been changing around the import script to work more flexibly as a class with fewer opinions about what it's fetching. It should integrate well with these changes. If there's time on the call today we can talk that over quickly.

Reply:
awesome! 👍